icons/flutter.svg

```diff
@@ -0,0 +1,13 @@
+<svg width="960" height="960" viewBox="0 0 960 960" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
The icon should be `16x16` in size and [should have some empty space around it](https://github.com/material-extensions/vscode-material-icon-theme/blob/main/CONTRIBUTING.md#icon-spacing)